specs2/specs2.bzl

```diff
@@ -5,84 +5,28 @@ load(
     _extract_major_version = "extract_major_version",
     _scala_mvn_artifact = "scala_mvn_artifact",
 )
-load(
-    "@io_bazel_rules_scala//scala:scala_maven_import_external.bzl",
-    _scala_maven_import_external = "scala_maven_import_external",
-)
+load("//third_party/repositories:repositories.bzl", "repositories")
 
 def specs2_version():
     return "4.4.1"
 
 def specs2_repositories(
         scala_version = _default_scala_version(),
-        maven_servers = _default_maven_server_urls()):
+        maven_servers = _default_maven_server_urls(),
+        overriden_artifacts = {}):
     major_version = _extract_major_version(scala_version)
```

Comment:
Maybe this `major_version = _extract_major_version(scala_version)` could be in `repositories` function? It would be possible to remove this duplicated line from other `*_repositories`